README.md

```diff
@@ -261,13 +261,59 @@ const { _, isError } = useForm(/* ... */)
 isError /* true / false */
 ```
 
+3. Use `&nbsp;` to avoid height collapse of `<p>` when there is no messages
+
+#### Default message for form
+
+Use `defaultMessage`sq to define placeholders for form field validation information.  The default value is `''`,you can set it to `u00A0` , which will be escaped to `&nbsp` during rendering , to avoid the height collapse problem of '<p>' when there is no messages.  
```

Comment:
```suggestion
Use `defaultMessage` to define a placeholder for the form field validation error message. The default value is `''`, you can set it to `u00A0`, which will be escaped to `&nbsp` during rendering, to avoid the height collapse problem of '<p>' when there is no messages.  
```

Reply:
感谢指导，已检查并订正

Reply:
辛苦了